operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +40,32 @@ Below is how SET prioritizes what is worked on.
+2. OpsGenie: Support has an e-mail address they will send urgent escalations to. These result in pages in OpsDuty, first sent to the SET Primary, then to SET Back-up and ultimately to the SET Lead when not timely acknowledged.
+3. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
+4. The [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel. This is the place where support and community can ask any type of R&D related questions.
+
+## Workflow
+When a new issue is raised, follow these steps:
+
+1. **Create a JIRA ticket** for the issue, initially connected to the "Sustained Engineering" team, and assigned to the SET Primary and moved to _In Progress_. Ensure the ticket describes the problem clearly and is as specific as possible. For example, if Mattermost is slow specify what is slow: the server, the client, a particular request.
+2. **Understand the customer impact and priority of the issue.** Determine the priority of the issue following [the levels specified here](https://support.mattermost.com/hc/en-us/articles/360038546151-Ticket-Priority-Guidelines), clearly mark this priority in the ticket. If the issue is really a feature request or low-urgency bug, change the ticket type and reassign it to the appropriate team. In that case, the SET responsibilities end. 
+4. If the issue is of **L1 (critical) or L2 (major), create a run of the [SET Incident Playbook](https://community-daily.mattermost.com/playbooks/playbooks/w5qizi5kojd7jejrsfpat5dupo)** and follow the playbook. Make sure that updates (using the Playbooks "Post update" feature) are posted **at least daily**.
+3. **Investigate the source of the issue,** update the JIRA ticket accordingly. Ensure the following things are clearly listed in the ticket (where appropriate):
+   * Mattermost version
+   * Database type and version
+   * Configuration used
+   * Relevant logs
+   * A CPU profile (in case of high-CPU usage issue)
+   * A heap profile (in case of memory use)
+   * Stack trace (in case  a service gets stuck)
+4. Hand-over (optional): once the issue is sufficiently narrowed down _and_ there is a more appropriate owner than SET, hand the issue over as follows:
+   * If a SET Incident playbook was run, follow the hand-over steps in that playbook
+   * Reassign the JIRA ticket to the new owning team
+
```

Comment:
I'm missing stating what's expected from Primary once he starts the incident. He is supposed to command the incident, but what does that mean to them?
It doesn't have to be on this page, though. We could create a shorter version with the work so it can be quickly read by anyone (say an eng. is back to SET but he hasn't done this for months, what's the minimum he needs to remember the process?)